File: hanoi-tower/hanoi_tower.py
def tower(disc, a, b, c):
    if disc < 1:
        return
    tower(disc-1, a, c, b)
    print('Move disc {} from peg {} to peg {}'.format(disc, a, c))
    tower(disc-1, b, a, c)


def main():

    tower(3, 'A', 'B', 'C')


if __name__ == '__main__':
    main()

File: hanoi-tower/hanoiTower.js
function tower(disc, a, b, c) {
  if (disc < 1) return;

  tower(disc - 1, a, c, b);
  console.log('Move disc ' + disc + ' from peg ' + a + ' to peg ' + c);
  tower(disc - 1, b, a, c);
}

tower(3, 'A', 'B', 'C');